### PR TITLE
feat: Add container spec sys_cpu_online

### PR DESCRIPTION
**Pull request by wushiqinlou**
Signed-off-by: jiazhang <jiazhang@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:
This spec is used to add container check on one insights-plugins advisor rule

### CODE REVIEW

**Review by xiangce — CHANGES_REQUESTED** [review]
This one seems to be duplicated with #3611, which one do you prefer to keep? And please get the spec approval at first.

**Comment by wushiqinlou** [comment]
@xiangce - Please review.

File: insights/parsers/cpu_online.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""
+cpu_online - File ``/sys/devices/system/cpu/online``
+=======================================================
```
Comment (xiangce) [inline insights/parsers/cpu_online.py:3]: Please modify the length of this line?

Comment (wushiqinlou) [reply insights/parsers/cpu_online.py]: @xiangce - Updated, please check

**Review by xiangce — APPROVED** [review]

**Pull request merged**